decidim-core/lib/decidim/content_renderers/hashtag_renderer.rb

```diff
@@ -29,8 +29,9 @@ class HashtagRenderer < BaseRenderer
       # @return [String] the content ready to display (contains HTML)
       def render(links: true, extras: true, html_escape: false, strip_tags: false)
         text = content
-        text = strip_tags(text) if strip_tags
+        text = sanitize(text, tags: %w(strong em a)) if strip_tags
         text = decidim_html_escape(text) if html_escape
+        text = Anchored::Linker.auto_link(text)
```

Comment:
As @andreslucena suggested, we should add target blank and rel noopener:

```suggestion
        text = Anchored::Linker.auto_link(text, target: "_blank", rel: "noopener")
```

Reply:
Why is this added to the hashtag renderer? Shouldn't we move it to `decidim_html_escape`? We're already calling `sanitize` there.

Reply:
Good question, why is the HashtagRenderer performing the striping/sanitization of the body of the Proposal and not the ProposalPresenter?

I've just added it here because it is the place where the body of the proposal strips_tags.

Also `decidim_html_escape` is not forced by the presenters, and its default value is false... instead the presenters tell the HashtagRenderer to `strip_tags`
https://github.com/decidim/decidim/blob/4dc1cc086396c5b860efebefb28be8f4fee50775/decidim-proposals/app/presenters/decidim/proposals/proposal_presenter.rb#L50
https://github.com/decidim/decidim/blob/4dc1cc086396c5b860efebefb28be8f4fee50775/decidim-proposals/app/presenters/decidim/proposals/collaborative_draft_presenter.rb#L36

A small improvement may be tell the HashtagRenderer to sanitize with `decidim_html_escape` instead of `strip_tags`. I'll proceed

Reply:
@oriolgual  By the way, I see that `decidim_html_escape` delegates to `ERB::Util.unwrapped_html_escape(text.to_str)` which is documented as "not for public consumption! Seriously!": https://apidock.com/rails/ERB/Util/unwrapped_html_escape/class

Do you agree to replace `ERB::Util.unwrapped_html_escape(text.to_str)` for a call to Rails' sanitizer similar to: `sanitize(text, tags: %w(a), attributes: %w(target rel))'

Reply:
Sorry, I did not catch what `decidim_html_escape` was really escaping the tags to the form `&amp;lt;a ...` so I'm not going to change it. But I think you should really consider alternatives to delegating to ERB.

Reply:
I've discarded keeping existing html anchors, but sanitized. Maybe it would be better to work it out in its own specific issue.

Reply:
Sorry, I meant `decidim_sanitize` instead of `decidim_html_escape`. What worries me about adding it to `HashtagRenderer` are two things:

1. I wouldn't expect a class with this name to also parse URLs, maybe we could just rename it to LinkRenderer?
2. We use `decidim_sanitize` in many more places than `HashtagRenderer`, so it can be confusing if you add an URL to some content and sometimes it gets rendered as a link and sometimes not.

Reply:
I've also checked and we're barely using the `html_escape` option from the hashtag renderer, maybe we could move it to the presenter so `HashtagRenderer` does less things.

Reply:
The same goes for `strip_tags`

Reply:
Hi @oriolgual I've discarded the sanitization. Probably it should be addressed more Decidim widely.